### PR TITLE
Updated CreateTaskDialog.tsx

### PR DESCRIPTION
**Pull request by Joshua-Vankayalapati**
Auto-Create State: autoCreate state added to track whether the checkbox is checked.
Checkbox Component: Simple checkbox component is rendered before the "Cancel" button.
Conditional Close: If "Auto-Create" is checked, the modal will remain open, even after the task is created.
Auto-Create feature

### CODE REVIEW

File: src/components/TaskForm/CreateTaskDialog.tsx [hunk 1]
```diff
@@ -47,11 +48,27 @@ export function CreateTaskDialog({ projectId }: Props) {
...
+    <div className="flex items-center gap-2">
+      <input
+        type="checkbox"
+        id="autoCreate"
+        checked={autoCreate}
+        onChange={(e) => setAutoCreate(e.target.checked)}
+      />
+      <label htmlFor="autoCreate">Auto-Create</label>
+    </div>
+  )
```
Comment (Balastrong) [inline src/components/TaskForm/CreateTaskDialog.tsx:70]: There's already a Checkbox component in `src/components/ui`, let's use that one :)

**Review by Balastrong — COMMENTED** [review]
Thanks for the PR! A couple of extra thoughts:

1) Can you remove the inline comments? Commenting the code is fine but only if you need to explain something that isn't clear in the code. For example a useState should be already understandable by the name of the variables, no need to comment it.

2) After the submit with the checkbox enabled we should also reset the form. Do you want to do it on this PR?

Thanks!

File: src/components/TaskForm/CreateTaskDialog.tsx (hunk 1 above)
Comment (Joshua-Vankayalapati) [reply src/components/TaskForm/CreateTaskDialog.tsx:70]: Sure! I'll replace the current checkbox with the one from src/components/ui for consistency. I'll also handle the form reset after submission when the checkbox is enabled and include it in this PR.

**Comment by albertasaftei** [comment]
the label should be renamed. I wasn't understanding what was it doing before seeing the demo.

Something like "Create another task". With this label the modal is too short but making it larger to have enough space it wouldn't be a problem imo

File: src/components/TaskForm/CreateTaskDialog.tsx (hunk 1 above)
Comment (Joshua-Vankayalapati) [reply src/components/TaskForm/CreateTaskDialog.tsx:70]: Sorry, I was unable to make any changes due to this issue still be persisted whenever I try to launch the project.
I followed the instructions as per README.md file. 

Tried different ways to resolve this error. Help pls. 
Any help is good.
projectruntime error
projview

Comment (Balastrong) [reply src/components/TaskForm/CreateTaskDialog.tsx:70]: Have you tried to logout and login again on the browser?

I don't think it's an issue in your code anyway

**Comment by Joshua-Vankayalapati** [comment]
Understood. I have updated the label for the "Auto-Create" checkbox to "Create Another Task" as requested.

Key updates include:

The form fields will reset after task creation, allowing users to input new information seamlessly.
With the checkbox enabled, the dialog box will remain open, enabling users to create additional tasks without closing the dialog.
The checkbox and its label now adhere to the project's existing UI style.

Please note:
The status of the checkbox will revert to undefined after task creation, but the previous status will be displayed. Users will need to reselect their option. Additionally, extending the dialog box width and refining these features may require further development from someone with advanced React and TypeScript experience.


enabled open dialog-box

File: src/components/TaskForm/CreateTaskDialog.tsx (hunk 1 above)
Comment (Joshua-Vankayalapati) [reply src/components/TaskForm/CreateTaskDialog.tsx:70]: > Have you tried to logout and login again on the browser?
> 
> I don't think it's an issue in your code anyway

Yes, that worked. Thank you

**Comment by Balastrong** [comment]
That's great, thanks!

I think this PR got closed automatically when you deleted the repo and is no longer in sync with your code, could you please open a new PR? :)

**Comment by Joshua-Vankayalapati** [comment]
> That's great, thanks!
> 
> I think this PR got closed automatically when you deleted the repo and is no longer in sync with your code, could you please open a new PR? :)

Ok, sure

**Pull request closed without merging**